.github/scripts/changed_files.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Get the list of changed files
+changed_files=$(git diff --name-only origin/$GITHUB_BASE_REF)
+
+# Define the directories to check
+allowed_directories=("docs/" "ui/" "website/" "grafana/" ".github/")
```

Comment:
- `allowed_directories` -> `skipped_directories`
- Shall we exclude ".github/" from the skipped directories since changes made to github workflow may break things

Reply:
Hmm okay. Works for me